tools/publication.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
```

Comment:
why `env sh` and not `bash`

Reply:
Nice spot, `#!/usr/bin/env sh` is what we have most. `get_version_next.sh` and `get_version_released.sh`are leftovers from development time that I should not have committed.